runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -4193,6 +4241,8 @@ void amReqFn_msgOrdFence(c_nodeid_t node,
     haveAmosOut = (tcip->amoVisBitmap != NULL
                    && bitmapTest(tcip->amoVisBitmap, node));
     break;
+
+  // XXX anything to do here for FAMO result?
```

Comment:
No, shouldn't be anything to do. This is making sure that any initiator-side stores that the MCM requires to be ordered before an atomic op that might modify memory are force to become visible before that atomic op is fired. Whether or not the atomic op is fetching doesn't matter.